internal/connector/pingone/pingone_platform_connector.go

```diff
@@ -0,0 +1,109 @@
+package pingoneplatformconnector
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/patrickcping/pingone-go-sdk-v2/pingone"
+	"github.com/pingidentity/pingctl/internal/connector"
+	connectorcommon "github.com/pingidentity/pingctl/internal/connector/common"
+	platformresources "github.com/pingidentity/pingctl/internal/connector/pingone/resources/pingoneplatform"
+	"github.com/pingidentity/pingctl/internal/logger"
+)
+
+const (
+	serviceName = "pingone-platform"
+)
+
+// Verify that the connector satisfies the expected interfaces
+var (
+	_ connector.Exportable      = &PingonePlatformConnector{}
+	_ connector.Authenticatable = &PingonePlatformConnector{}
+)
+
+type PingonePlatformConnector struct {
+	clientInfo connector.SDKClientInfo
+}
+
+// Utility method for creating a PingonePlatformConnector
+func Connector(ctx context.Context, apiClient *sdk.Client, exportEnvironmentID string) *PingonePlatformConnector {
```

Comment:
If this is going to be at the same level as future connectors, this method will need to be renamed (maybe PlatformConnector)